rules/aep0004/resource_name_components_alternate.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/jhump/protoreflect/desc"
 )
 
-var identifierRegexp = regexp.MustCompile("^{[a-z][_a-z0-9]*[a-z0-9]}$")
+var identifierRegexp = regexp.MustCompile("^{[a-z][-_a-z0-9]*[a-z0-9]}$")
```

Comment:
```suggestion
var identifierRegexp = regexp.MustCompile("^{[a-z][-a-z0-9]*[a-z0-9]}$")
```
I think we shouldn't allow underscores then? it must be kebab-case.